# Desired force in newtons for each of the six thrusters.
#
# When looking at the ROV from above, the configuration looks like this:
#
# |‾‾‾|__|‾‾‾| FRONT END
# |F1      F2|
# |    F5    |
# |          |
# |    F6    |
# |F4      F3|
# |__________| REAR END



# Front left thruster, positive forward
float64 F1

# Front right thruster, positive forward
float64 F2

# Rear left thruster, positive backward
float64 F3

# Rear right thruster, positive backward
float64 F4

# Front vertical thruster, positive down
float64 F5

# Rear vertical thruster, positive down
float64 F6
